### ISSUE TITLE
[Bug]: Missing dialog for selecting images with Android (Google) photo app

### ISSUE THREAD

**Issue by OpenGreenStreet** [commons-app/apps-android-commons#5818]
### Summary

Missing dialog for selecting images with Android (Google) photo app

### Steps to reproduce

When adding images via the Commons app, the (Google) Photo app was launched in earlier versions - as far as I remember.
Currently, a list view is offered in the Camera folder. This makes sense in some cases. In most cases, I think it would be much more convenient if the Photos app were at least optionally available for selecting images.

### Expected behaviour

The Android (Google) photo app is optionally available (as with the Wikimedia Upload Wizard)

### Actual behaviour

Currently, a list view is offered in the Camera folder. 

### Device name

Google Pixel Pro 7

### Android version

Android 14

### Commons app version

5.0.2~05ffd123e

### Would you like to work on the issue?

None

**Comment by nicolas-raoul** [comment]
This might be related to the "Use document based photo picker" setting. Would you mind trying with it enabled and disabled?

**Comment by OpenGreenStreet** [comment]
@nicolas-raoul ,

Yes, that fits. Thank you! I have now switched off the option (“Use document based photo picker”). The location information has nevertheless been retained. Both options have their advantages. I'll see which variant I like better in the long term. Probably the selection via the (Google) photo app.

PS: Perhaps the (Google) photo app could be referred to in the explanation. The connection wasn't immediately clear to me...
